content/reference/endpoints/inbox_conversations.md

```diff
@@ -43,6 +44,7 @@ closed_at            | [Time](/reference/enums#formats) | Read       | Time when
 deadline_response_at | [Time](/reference/enums#formats) | Read/Write | Conversation's deadline response time.
 last_message_at      | [Time](/reference/enums#formats) | Read       | Conversation's last message time.
 marked_as_spam_at    | [Time](/reference/enums#formats) | Read/Write | Time when Conversation was marked as spam.
+read_at              | [Time](/reference/enums#formats) | Read       | Conversation's read time.
```

Comment:
Time when conversations was viewed ?

viewed_at would be a better name btw :(

Reply:
IMO `read` better match with current field

Reply:
the problem with `read` is that it has the same form in present and past tense. So it can be read as a command :) Which is confusing. `viewed_at` can't be misinterpreted.